linera-views/src/backends/lru_caching.rs

```diff
@@ -204,17 +205,17 @@ where
         &self,
         keys: Vec<Vec<u8>>,
     ) -> Result<Vec<Option<Vec<u8>>>, Self::Error> {
-        let Some(lru_read_values) = &self.lru_read_values else {
+        let Some(cache) = &self.cache else {
```

Comment:
Why sometimes we have `let Some(cache) = ...` and sometime we have `let Some(values) = ...`?

Reply:
good catch